packages/asset-swapper/src/utils/market_operation_utils/curve_utils.ts

```diff
@@ -0,0 +1,9 @@
+import * as _ from 'lodash';
+
+import { MAINNET_CURVE_CONTRACTS } from './constants';
+
+export const getCurveAddressesForPair = (takerToken: string, makerToken: string): string[] => {
+    return Object.keys(
+        _.pickBy(MAINNET_CURVE_CONTRACTS, tokens => tokens.includes(takerToken) && tokens.includes(makerToken)),
```

Comment:
nit: possible to use native Array.filter instead of lodash?

Reply:
we need to filter out keys based on their values, is there a way to do that with Array.filter?